requirements/test.txt

```diff
@@ -1,4 +1,3 @@
 -r base.txt
 mock==1.3.0
-tox==2.1.1
```

Comment:
You only need tox if you're actually running the tests. Travis [uses](https://github.com/cfpb/cfgov-refresh/blob/master/.travis.yml#L34) `cfgov.settings.test` when running them.